File: python-beginner-workbook/module-02-basics/examples/example-03-calculations.py
# example-03-calculations.py
# Demonstrates calculations and f-string formatting

# Get order details from user
item_name = input("Enter item name: ")
base_price = float(input("Enter base price: £"))
quantity = int(input("Enter quantity: "))

# Calculate subtotal
subtotal = base_price * quantity

# Calculate VAT (20%)
vat_rate = 0.20
vat_amount = subtotal * vat_rate

# Calculate total
total = subtotal + vat_amount

# Display formatted order summary
print("\n=== Order Summary ===")
print(f"Item: {item_name}")
print(f"Base Price: £{base_price:.2f}")
print(f"Quantity: {quantity}")
print(f"Subtotal: £{subtotal:.2f}")
print(f"VAT (20%): £{vat_amount:.2f}")
print(f"{'='*25}")
print(f"Total: £{total:.2f}")

File: csharp-beginner-workbook/module-02-basics/examples/example-03-calculations.cs
// Example 3: Calculations and String Interpolation
// This demonstrates how to perform calculations and format output

Console.Write("Enter the price: $");
string priceText = Console.ReadLine();
double price = double.Parse(priceText);

Console.Write("Enter the quantity: ");
string quantityText = Console.ReadLine();
int quantity = int.Parse(quantityText);

double subtotal = price * quantity;
double tax = subtotal * 0.20;  // 20% tax
double total = subtotal + tax;

Console.WriteLine("\n=== Order Summary ===");
Console.WriteLine($"Price: ${price:F2}");
Console.WriteLine($"Quantity: {quantity}");
Console.WriteLine($"Subtotal: ${subtotal:F2}");
Console.WriteLine($"Tax (20%): ${tax:F2}");
Console.WriteLine($"Total: ${total:F2}");

